setup.py

```diff
@@ -52,18 +53,28 @@ def package_files(directory):
         "entrypoints",
         # Pin sqlparse for: https://github.com/mlflow/mlflow/issues/3433
         "sqlparse>=0.3.1",
+        # Required to run the MLflow server against SQL-backed storage
         "sqlalchemy<=1.3.13",
         "gorilla",
         "prometheus-flask-exporter",
     ],
     extras_require={
         "extras": [
             "scikit-learn; python_version >= '3.5'",
-            "boto3>=1.7.12",
-            "mleap>=0.16.0",
-            "azure-storage-blob>=12.0",
+            "scikit-learn==0.20; python_version < '3.5'",
```

Comment:
Can we remove this line because we don't support python < 3.5 anymore?

Reply:
Yep! Thanks.